package.json

```diff
@@ -40,7 +40,7 @@
 		"eslint": ">=6.0.0",
 		"rollup": "^2",
 		"sourcemap-codec": "1.4.8",
-		"svelte": "^3.2.0",
```

Comment:
🤔 Master test are broken with svelte "3.35.0". This is a temporary fix just to be sure all the tests working fine.

Reply:
I open an issue for that #94